l10n_it_ricevute_bancarie/models/account/account.py

```diff
@@ -130,13 +131,33 @@ def _compute_is_unsolved(self):
             if len(invoice.unsolved_move_line_ids) != reconciled_unsolved:
                 invoice.is_unsolved = True
 
-    _inherit = "account.invoice"
     unsolved_move_line_ids = fields.Many2many(
         'account.move.line', 'invoice_unsolved_line_rel', 'invoice_id',
         'line_id', 'Past Due Journal Items')
     is_unsolved = fields.Boolean(
         "Is a past due invoice", compute="_compute_is_unsolved", store=True
     )
+    is_riba_payment = fields.Boolean(
+        'Is Riba Payment', related='payment_term_id.riba', default=False
+    )
+    riba_partner_bank_id = fields.Many2one(
+        'res.partner.bank',
+        string='RiBa Bank Account',
+        help='Bank Account Number to which the RiBa will be debited. '
+        'If not set,fisrt bank in partner will be used.',
```

Comment:
Per uniformità con i termini già esistenti.

```suggestion
        'Is C/O Payment', related='payment_term_id.riba', default=False
    )
    riba_partner_bank_id = fields.Many2one(
        'res.partner.bank',
        string='C/O Bank Account',
        help='Bank Account Number to which the C/O will be debited. '
        'If not set, first bank in partner will be used.',
```